exercises/vecs/vecs1.rs

```diff
@@ -18,7 +17,7 @@ fn array_and_vec() -> ([i32; 4], Vec<i32>) {
     // v.push(30);
     // v.push(40);
 
-    // let v: Vec<i32> = Vec::from(a);
+    let v: Vec<i32> = Vec::from(a);
```

Comment:
Heard of [`slice::to_vec()`](https://doc.rust-lang.org/std/primitive.slice.html#method.to_vec)?

Reply:
Also: Does "from" take a `&Vec` (ref) or does it take ownership of a vector? You pass it `a` but you use `a` later. Can you explain this?

Reply:
I find it more readable, but just so you know both options exist.